server/app/assets/javascripts/admin_predicate_configuration.ts

```diff
@@ -206,44 +206,28 @@ class AdminPredicateConfiguration {
       operatorDropdownContainer.dataset.questionId,
     )
 
+    // Each value input has its own help text
     const csvHelpTexts = document.querySelectorAll(
       `#predicate-config-value-row-container [data-question-id="${questionId}"] .cf-predicate-value-comma-help-text`,
     )
-
-    // The help text div is present for inputs that allow specifying
-    // multiple values in a single text input. It won't be present
-    // for other input types e.g. multiselect inputs.
-    if (csvHelpTexts != null) {
-      const shouldShowCommaSeperatedHelpText =
+    csvHelpTexts.forEach((div: Element) =>
```

Comment:
praise:  This is so much clearer and more succinct.  Thanks for cleaning up!

Reply:
Thank you! :)